react/features/large-video/components/styles.js

```diff
@@ -1,13 +1,36 @@
 import { StyleSheet } from 'react-native';
 
-import { ColorPalette, createStyleSheet } from '../../base/styles';
+import { BoxModel, ColorPalette, createStyleSheet } from '../../base/styles';
 
 /**
  * Size for the Avatar.
  */
 export const AVATAR_SIZE = 200;
 
 export default createStyleSheet({
+    /**
+     * View that contains the indicators.
+     */
+    indicatorContainer: {
+        flex: 1,
+        flexDirection: 'row',
+        margin: BoxModel.margin,
+        position: 'absolute',
+        right: 0,
+
+        // Both on Android and iOS there is the status bar which may be visible.
+        // On iPhone X there is the notch. In the two cases BoxModel.margin is
+        // not enough.
+        top: BoxModel.margin * 3
+    },
+
+    /**
+     * Indicator container for wide aspect ratio.
+     */
+    indicatorContainerWide: {
+        right: 80
```

Comment:
if this value is not relative to anything please declare a constant for it like the AVATAR_SIZE above